go.mod

```diff
@@ -20,8 +20,8 @@ require (
 	github.com/multiversx/mx-chain-logger-go v1.0.13
 	github.com/multiversx/mx-chain-scenario-go v1.2.1
 	github.com/multiversx/mx-chain-storage-go v1.0.13
-	github.com/multiversx/mx-chain-vm-common-go v1.5.5
-	github.com/multiversx/mx-chain-vm-go v1.5.16
+	github.com/multiversx/mx-chain-vm-common-go v1.5.6-0.20231025104250-d24edf1e19ae
```

Comment:
let's not forget about proper release

Reply:
I will add proper tags after system test

Reply:
added